src/promptflow-core/promptflow/core/_serving/flow_invoker.py

```diff
@@ -252,7 +251,12 @@ async def invoke_async(self, data: dict, run_id=None, disable_input_output_loggi
             data, run_id=run_id, disable_input_output_logging=disable_input_output_logging
         )
         # Get base64 for multi modal object
-        resolved_outputs = self._convert_multimedia_data_to_base64(result)
+        output_dict = convert_eager_flow_output_to_dict(result.output)
```

Comment:
@D-W- Please double confirm here

Reply:
Original code was added in sync invoker. seems this one was added to async one

Reply:
yes, for fastapi we will all change to use this async api, so this is to make sure the logic is aligned with the sync call